kerchunk/kerchunk-in-practice.ipynb

```diff
@@ -0,0 +1,2704 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            h5chunks = SingleHdf5ToZarr(infile, input_file, inline_threshold=300)
</pre><p>what does the threshold do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
updated with comment